### PR TITLE
[OFFAPPS-611] Ensure HTML text in original description is sent correctly when creating child ticket

### PR DESCRIPTION
**Pull request by pmgrove**
:koala:

check for this.ticket().comment.().useRichText() to ensure HTML is sent correctly when creating a ticket via the api

/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-611

### Risks
 - None

### CODE REVIEW

**Comment by danielbreves** [comment]
:+1:

File: app.js [hunk 1]
```diff
@@ -427,12 +428,22 @@
...
         "custom_fields": [
-          { id: this.ancestryFieldId(), value: 'child_of:' + this.ticket().id() }
+          {
+            id: this.ancestryFieldId(),
+            value: 'child_of:' + this.ticket().id()
+          }
         ]
       };
 
+      if (this.ticket().comment().useRichText()){
```
Comment (adammw) [inline app.js]: tossing up if it's better to do this or leave the comment out of the larger params and just set `params.comment[this.ticket().comment().useRichText() ? 'html_body' : 'body'] = this.formDescription();`

**Comment by xuebinglee** [comment]
:+1:

**Comment by adammw** [comment]
:+1:

**Comment by pmgrove** [comment]
:haircut: @adammw

**Pull request merged**